lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -41,20 +43,30 @@ multi sub MAIN('web', ExistingFile $cfg-filename, ExistingFile $model-filename,
 #| Run the model
 multi sub MAIN('run', ExistingFile $filename, ExistingFile $input, Str $tech-file?,
                SupportedLanguage :$language = 'de', Str :$prints = 'All', Str :$variants = 'SHL',
-               Bool :$csv, Bool :$include-filters, Int :$batch=1, Int :$degree=4, Int :$max-runs
+               Bool :$csv, Bool :$include-filters, Int :$batch=1, Int :$degree=4, Int :$max-runs,
+               OutputFormat :$format = 'txt'
```

Comment:
s t rlly wrth abbrvtng txt? :-)

Reply:
Which ones, `max-runs` and `tech-file`? :-)

Reply:
No, t**e**xt. :-)